router/core/graphql_prehandler.go

```diff
@@ -132,6 +162,9 @@ func (h *PreHandler) Handler(next http.Handler) http.Handler {
 
 		statusCode = ww.Status()
 		writtenBytes = ww.BytesWritten()
+
+		// Evaluate the request after the request has been handled by the engine
+		hasRequestError = requestContext.hasError
```

Comment:
@fiam in order to do this I had to move the creation of the context before the graphql handler.